conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/platform/data/tezos/generic/TezosDataEndpoints.scala

```diff
@@ -1,42 +1,16 @@
-package tech.cryptonomic.conseil.api.routes.platform.data.tezos
+package tech.cryptonomic.conseil.api.routes.platform.data.tezos.generic
```

Comment:
The purpose is to have classes that are "specific-implementation" independent. For me it was a bit hard to find out, when (in which classes) we are wiring up our `endpoints` definitions with Akka and Circe. I believe the reason behind `Endpoints` was also to add thing "generalization", so we could at some point for example replace Akka with http4s. Classes that are in `.generic` package are Akka and circe independent. Classes that are one level higher, can use (and they are using) Akka + Circe in it's implementations. Do you think it make sense to have separation like that, or I should put all-in-one?

Reply:
I think the `generic` term was referring to the "generic chain" concatenation, as code that is "platform-agnostic".

I'm not sure we want to invest too much into optimize for backend-agnostic code, and do such kind of technical-modularity, as opposed to domain-modularity. We're not building a generic library and the time to refactor for different backends is not that much, so I'm wondering if this would pay off or complicate matters fruitlessly.

What do you think?

Reply:
Well, I did it, because it was easier for me to understand that. I don't think we will have more than one json parser integrated in the code, so I can change it back (simply move everything from generic, one level higher). No problem at all :)